### DISCUSSION TITLE
Remove accuse and add a notes panel instead

### DISCUSSION THREAD

**Discussion by simonhackler** [bromberry-games/Inkvestigations#150] [Game design]
I think we should remove accuse because it limits us to do only mysteries of a certain kind.
I am sure it is possible to have the accuse prompt in a way where we can do arbitrary mysteries with how many culprits we want.
(0,1,2 or whatever)

Then instead of the accuse panel I think we should have a notes panel, which shows the most important characters and allows users to add notes. We could have notes for every character or just one big note. 

In the accuse panel we would also have a solve button.
We could setup solve in a way where players have to provide motive, opportunity etc in separate inputs.
This could force players to think about the mysteries the way we want them to.

**Comment by zivky** [comment]
I think that is great, turn it into a information window with all the relevant information for the case. I agree that the way accuse currently works is a bit too constraining. A solve button where you need to enter everything individually (perpetrator, motive, opportunity, evidence) works well because it's easier for gpt (probably) and it is a constraint that helps players progress. For example, if they think they have solved it, but are missing opportunity, they know what to ask next.
 
As for the notes: Shoutout to OrionFive on reddit for giving us feedback that it would be cool to have notes for each suspect. The advantage of individual notepads is that you would have inherent organization. However, we would need to add one general notepad for information that isn't connected to any character yet. What would be the disadvantages? Off the top of my head, it sounds like a lot to keep track of ironically. For example, if you know some character has some info but you forgot which, you need to click through all the notes. Though that sounds like it should be a rare case. But what if you want to compare info between to characters, you need to click back and forth. Overall, I think having individual ones with a general one would be more fun because you get the option to be systematic if you want.
